src/Type.h

```diff
@@ -425,6 +425,18 @@ struct Type {
         return code() == Handle;
     }
 
+    // Returns true iff type is an integral type where overflow is defined.
```

Comment:
change comment from "an integral" to "a signed integral"

Reply:
fixed in 4f0a437